CLAUDE.md

```
@@ -1,378 +1,1438 @@
+      - name: Test
+        run: pytest --cov=. --cov-report=xml
+      - name: Upload coverage
+        uses: codecov/codecov-action@v3
 ```
 
 ---
 
-## 🔐 Security
+## 📝 Commit Message Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)